website/docs/components/modal/partials/code/component-api.md

```diff
@@ -18,6 +18,9 @@
   <C.Property @name="color" @type="enum" @values={{array "neutral" "warning" "critical" }} @default="neutral">
     Sets the color scheme for the Modal Header elements: icon, tagline, and title.
   </C.Property>
+  <C.Property @name="returnFocusTo" @type="string">
```

Comment:
Let me know if you think I should add also an example in the "how to use" section.